README.md

```diff
@@ -300,22 +300,22 @@ Now that you have a copy of our official board, you just need to pass the cards
 
 We also have **labels** to help you have more control through the process. The meaning of each of these labels is:
 
-- `Main Curriculum`: cards with that label represent courses that are listed in our curriculum. 
+- `Main Curriculum`: cards with that label represent courses that are listed in our curriculum.
 - `Extra Courses`: cards with that label represent courses that was added by the student.
 - `Doing`: cards with that label represent courses the student is current doing.
 - `Done`: cards with that label represent courses finished by the student. Those cards should also have the link for at least one project/article built with the knowledge acquired in such course.
 - `Section`: cards with that label represent the section that we have in our curriculum. Those cards with the `Section` label are only to help the organization of the Done column. You should put the *Course's cards* below its respective *Section's card*.
 - `Extra Sections`: cards with that label represent sections that was added by the student.
 
-The intention of this board is to provide for our students a way to track their progress, and also the ability to show their progress through a public page for friends, family, employers, etc. You can change the status of your board to be **public** or **private**. 
+The intention of this board is to provide our students a way to track their progress, and also the ability to show their progress through a public page for friends, family, employers, etc. You can change the status of your board to be **public** or **private**.
 
 ### Should I take all courses?
 
 **Yes!** The intention is to conclude **all** the courses listed here!
 
 ### Duration of the project
 
-It may take longer to complete all of the classes compared to a  regular CS course, but I can **guarantee** you that your **reward** will be proportional to **your motivation/dedication**!
+It may take longer to complete all of the lessons compared to a regular CS course, but I can **guarantee** you that your **reward** will be proportional to **your motivation/dedication**!
```

Comment:
We can refactor the entire sentence to:

> It may take longer to complete all courses compared to a regular CS graduation, but I can **guarantee** you that your **reward** will be proportional to **your extra effort**!